stable/pipelines/README.md

```diff
@@ -157,6 +156,23 @@ rabbitmq:
 helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f values-external-rabbitmq.yaml
 ```
 
+### Using external Vault
+
+If you want to use external vault, set `vault.enabed=false` and create `values-external-vault.yaml` with below yaml configuration
+
+```yaml
+vault:
+  enabled: false
+global:
+  vault:
+    url: vault_url
+    token: vault_token
+```
+
+```bash
+helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f values-external-vault.yaml
+```
+
```

Comment:
please add external secret for Vault token example as well, as we have in Pipelines v1.4.4 chart

Reply:
Added the external secret example for vault